package tech.rket.auth.application.user.exception;

public class UserNotFoundException extends RuntimeException {
    public UserNotFoundException(Object value) {
    }
}
